Calculator/MainWindow.xaml

```diff
@@ -34,12 +34,17 @@
             </Style>
             <Style x:Key="ScreenContent" TargetType="Label">
                 <Style.Triggers>
-                    <Trigger Property="Content" Value="正无穷大">
+                    <Trigger Property="Content" Value="NaN">
```

Comment:
If there is a situation that the number showed on the screen label can not converted as a number, it may breaks the logic of the program, can you provide a example? I'll fix it.

Reply:
There is **not** a logical error in you program.
But if you calculate 0/0, you will find 'NaN' in your screen.
And if you calculate 1/0, you will find '∞' in your screen instead of '正无穷大'.
You set some triggers to make them become red.
However it not work as you think.
So, I change triggers' value to make your thought come true.